meson.build

```diff
@@ -39,3 +39,7 @@ import('pkgconfig').generate(
     name: 'Erfa',
     description: 'Essential Routines for Fundamental Astronomy',
 )
+
+if build_machine.system() != 'windows'
```

Comment:
Sorry I didn't have the time to look all details, but my question is: is the `dist` package supposed to be platform independent? I assume that we are talking about a source distribution.
In this case the file should be included in any case, including on `windows` platforms, correct?

Reply:
In theory yes, in practice autotools wasn't actually supported on Windows and can't generate a configure script there.

So the choices are probably:
- run build and test on Windows CI but not dist
- create different dists on Windows vs on other platforms.

I'm okay with either approach, so I can switch to the former.

Reply:
> In theory yes, in practice autotools wasn't actually supported on Windows and can't generate a configure script there.

OK understood.

I think that as soon as we distribute the complete source package (the one generated on unix) it is fine to keep the `dist` workflow also on windows.

... please feel free to proceed as you prefer.

Reply:
Let's drop the distcheck on Windows.